<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>851</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/5.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background :rgb(216, 216, 216);</string>
     </property>
     <property name="gridVisible">
      <bool>false</bool>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>欢迎使用本数字图像处理组合包


请在最上方的菜单栏选择自己想要的功能
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>祝您使用愉快</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>灰度化、均衡化操作</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>二值化</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="separator"/>
    <addaction name="actionotsu"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>图像的几何变换</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>图像分割</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>图像边缘检测</string>
    </property>
    <addaction name="actioncanny"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_13">
    <property name="title">
     <string>滤波器</string>
    </property>
    <addaction name="action3_3"/>
    <addaction name="separator"/>
    <addaction name="action3_4"/>
    <addaction name="separator"/>
    <addaction name="action5_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_13"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondsadsad">
   <property name="text">
    <string>dsadsad</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图像灰度化</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>图像均衡化</string>
   </property>
  </action>
  <action name="actioncanny">
   <property name="text">
    <string>canny</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>全局分割</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>动态分割</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>区域生长</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>分水岭</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>自选阈值</string>
   </property>
  </action>
  <action name="actionotsu">
   <property name="text">
    <string>otsu 算法</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>水平外凹</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>水平外凸</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>水平梯形形变</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>竖直梯形形变</string>
   </property>
  </action>
  <action name="action_31">
   <property name="text">
    <string>直角三角形形变</string>
   </property>
  </action>
  <action name="action_33">
   <property name="text">
    <string>等腰三角形形变</string>
   </property>
  </action>
  <action name="actionS">
   <property name="text">
    <string>S形变形</string>
   </property>
  </action>
  <action name="action_36">
   <property name="text">
    <string>图片旋转</string>
   </property>
  </action>
  <action name="action_38">
   <property name="text">
    <string>图片平移</string>
   </property>
  </action>
  <action name="actionx">
   <property name="text">
    <string>x 轴翻转</string>
   </property>
  </action>
  <action name="actiony">
   <property name="text">
    <string>y 轴翻转</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>3*3中值滤波</string>
   </property>
  </action>
  <action name="action3_4">
   <property name="text">
    <string>3*3均值滤波</string>
   </property>
  </action>
  <action name="action5_5">
   <property name="text">
    <string>5*5高斯滤波</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>立刻进行</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
